sycl/CMakeLists.txt

```diff
@@ -61,22 +68,30 @@ else()
 endif()
 
 if( NOT OpenCL_LIBRARIES )
-  message("OpenCL_LIBRARIES is missed. Try to build from GitHub sources.")
-  set(OpenCL_LIBRARIES "${LLVM_LIBRARY_OUTPUT_INTDIR}/libOpenCL.so")
+  message("OpenCL_LIBRARY is missed. Will try to download OpenCL ICD Loader from github.com")
```

Comment:
Changing OpenCL_LIBRARIES variable name to OpenCL_LIBRARY does not look right. "find_project(OpenCL)" uses module findOpenCL which sets OpenCL_LIBRARIES (and few other vars) if OpenCL library is found. OpenCL_LIBRARY is a cache var and can be used to tweak its behavior.
Same is true for similar change at line 50.

Reply:
Valery, the latest update to this patch does what you asked in e-mail, i.e. use CMAKE_STATIC_LIBRARY_SUFFIXCMAKE_STATIC_LIBRARY_SUFFIX/PREFIX for prefix/suffix:
`  if(MSVC)
    set(OpenCL_LIBRARIES
        "${LLVM_LIBRARY_OUTPUT_INTDIR}/${CMAKE_STATIC_LIBRARY_PREFIX}OpenCL${CMAKE_STATIC_LIBRARY_SUFFIX}")
    set(OPENCL_ICD_LOADER_WDK "-DOPENCL_ICD_LOADER_REQUIRE_WDK=OFF")
  else()
    set(OpenCL_LIBRARIES
        "${LLVM_LIBRARY_OUTPUT_INTDIR}/${CMAKE_SHARED_LIBRARY_PREFIX}OpenCL${CMAKE_SHARED_LIBRARY_SUFFIX}")
    set(OPENCL_ICD_LOADER_WDK "")
  endif()
`

I though not sure I understand what you ask/mean in your comment right above (about OpenCL_LIBRARIES vs OpenCL_LIBRARY).

Reply:
> I though not sure I understand what you ask/mean in your comment right above (about OpenCL_LIBRARIES vs OpenCL_LIBRARY).

Ok, finally I understood and fixed the message. Valery please re-review. Thank you!